content/applications/general/companies/digest_emails.rst

```diff
@@ -1,83 +1,232 @@
-recipients fields as needed.
+A separate page, with a blank digest email template appears, and presents a variety of editable
+settings, including:
 
-After you click **Save**, your new custom Digest Email is available as a selection in the **General
-Settings** dropdown menu.
+- :guilabel:`Digest Name`: the name of the digest email.
+- :guilabel:`Periodicity`: control how often digest emails are sent (:guilabel:`Daily`,
+  :guilabel:`Weekly`, :guilabel:`Monthly`, or :guilabel:`Quarterly`).
+- :guilabel:`Next Send Date`: the date on which the digest email will be sent again.
+- :guilabel:`KPIs` tab: check/uncheck each calculated :abbr:`KPI (key performance indicator)` that
+  appears in digest emails. A ticked box indicates an active :abbr:`KPI (key performance indicator)`
+  in the digest email. See the section on :ref:`digest-emails/kpis`.
+- :guilabel:`Recipients` tab: add/remove users who receive the digest emails. See the section on
+  :ref:`digest-emails/recipients`.
+
+From there, give the digest email a :guilabel:`Digest Name`, specify :guilabel:`Periodicity`,
+choose the desired :abbr:`KPIs (key performance indicators)`, and add :guilabel:`Recipients`, as
+needed.
+
+After clicking :guilabel:`Save`, the new custom digest email is available as a selection in the
+:guilabel:`Digest Email` field, located in the :menuselection:`Settings app --> Statistics section`.
 
 .. _digest-emails/custom-kpi:
 
-Custom KPIs with Studio
------------------------
+Custom KPIs with Odoo Studio
+============================
+
+The :abbr:`KPIs (key performance indicators)` on a digest email template form, in the
+:guilabel:`KPIs` tab, can be customized using Odoo *Studio*.
+
+.. warning::
+   Additional costs to the database subscription are incurred, should Odoo *Studio* need to be
+   installed.
+
+To begin, click the :guilabel:`🛠️ (tools)` icon in the top-right of the screen. This is the link to
+the Odoo *Studio* application.
 
-For either *Your Odoo Periodic Digest* or your own custom Digest Email, you can add your own KPIs
-by using Odoo Studio.
+In order to create additional fields, create two fields on the digest object:
 
-To begin, click the **Toggle Studio** icon or click the **Recipients** tab and then the ellipses
-:menuselection:`… icon --> Add Custom Field` to edit the template or add additional fields.
+#. Create a boolean field called `kpi_myfield`, and display it in the :guilabel:`KPIs` tab.
+#. Create a computed field called `kpi_myfield_value` that computes the customized :abbr:`KPI (key
+   performance indicator)`.
+#. Select the :abbr:`KPIs (key performance indicators)` in the :guilabel:`KPIs` tab.
 
-In order to create additional fields, you must create two fields on the digest object:
+.. tip::
+   Here is the `source code
+   <https://github.com/odoo/odoo/blob/aa09df25845f5497124185013bdb5706fcb2b082/addons/digest/models/digest.py>`_
+   for the `digest.py` file, which guides the programmer in the coding of the computed field.
```

Comment:
Since this doc is in 15.0, would it make sense to link the `15.0` version of the `digest.py`?
```suggestion
      <https://github.com/odoo/odoo/blob/15.0/addons/digest/models/digest.py>`_ for the `digest.py`
   file, which guides the programmer in the coding of the computed field.
```